webrender/res/cs_clip_border.glsl

```diff
@@ -155,8 +155,7 @@ void main(void) {
                                 clip_relative_pos);
 
     // Get AA widths based on zoom / scale etc.
-    vec2 fw = fwidth(local_pos);
-    float afwidth = length(fw);
+    float aa_range = 0.4 * length(fwidth(local_pos));
```

Comment:
can we have `0.4` as a constant defined somewhere with a proper name + comments?